src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
I tend to agree with you Oppen, it seems like something CMake should be doing.  From what I could see however, CMake is currently globbing all files recursively, so it will grab both files.  Im not too strong with CMake, so if thats a better aproach perhaps someone could assist?